src/less-style/inputs.less

```diff
@@ -76,3 +76,22 @@
         width: 100%;
     }
 }
+
+.fd-commander {
+    display: flex;
```

Comment:
When I remove all the flex styles the it looks terrible. Feel free to suggest alternate styles if you know of a better way.